keps/prod-readiness/sig-apps/2879.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2879
+beta:
+  approver: "@ehashman"
```

Comment:
Please change to myself [I have a context of changes around and Elana is out for August]

I will review later this week [though please work with SIG leads to have it approved first]

Reply:
Done